genienlp/data_utils/bootleg.py

```diff
@@ -373,20 +374,18 @@ def replace_features_inplace(self, examples, all_token_type_ids, all_tokens_type
         for n, (ex, tokens_type_ids, tokens_type_probs, tokens_qids) in enumerate(
             zip(examples, all_token_type_ids, all_tokens_type_probs, all_tokens_qids)
         ):
+            features = [Entity(*tup) for tup in zip(*[tokens_type_ids, tokens_type_probs, tokens_qids])]
```

Comment:
Why do you need *[ here?

Reply:
I don't know why I keep doing this!

Reply:
Should have a lint, catch all instances of *[, cause everytime I see it I get confused and think it's some obscure python thing!